### ISSUE TITLE
Bridge dataset action normalization

### ISSUE THREAD

**Issue by lyshuga** [octo-models/octo#12]
Hello,

Thank you for this amazing project. Great job!

I have a small question regarding the finetunning. In the code it is mentioned to use custom version of bridge dataset (https://rail.eecs.berkeley.edu/datasets/bridge_release/data/tfds/), there are some features just changed (e.g. names, action dimensions, etc.). I was wondering if there was any normalization applied to the action in your custom version of bridge dataset?

Also, one more question, in general in order to finetune your model should actions in new dataset be normalized at the stage of creation?

**Comment by kpertsch** [comment]
Hi, thanks for reaching out!

Our code is tested with the Bridge dataset version you linked, so best to use that one (all pre-trained checkpoints are using this data as well)!

The raw data does not have action normalization applied, but our data loader by default applies Gaussian normalization (ie 0-mean, unit variance) to both action and proprio inputs (see [here](https://github.com/octo-models/octo/blob/37951e4e6d708fd76374f6e09e716763fe2673b1/octo/data/dataset.py#L211)]

All our finetuning experiments used the same Gaussian action normalization, so best to use that default during finetuning too! But no need to pre-normalize the actions in your custom finetuning dataset, the data loader will take care of it!

**Comment by lyshuga** [comment]
Thank you for you answer 🙂
